packages/components/src/components/input/input.stories.ts

```diff
@@ -1,932 +1,363 @@
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'password' },
-              { type: 'attribute', name: 'label', value: 'Password' },
-              { type: 'attribute', name: 'password-toggle', value: true },
-              { type: 'attribute', name: 'placeholder', value: '.*' },
-              { type: 'attribute', name: 'help-text', value: 'use password display format' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'search' },
-              { type: 'attribute', name: 'label', value: 'Search' },
-              { type: 'attribute', name: 'placeholder', value: '^d{1,3}$' },
-              { type: 'attribute', name: 'help-text', value: 'use search format' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'tel' },
-              { type: 'attribute', name: 'label', value: 'Tel' },
-              { type: 'attribute', name: 'placeholder', value: '+49 1234 567891' },
-              { type: 'attribute', name: 'help-text', value: 'validate with tel format' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'text' },
-              { type: 'attribute', name: 'label', value: 'Text' },
-              { type: 'attribute', name: 'placeholder', value: '.*' },
-              { type: 'attribute', name: 'help-text', value: 'default type' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'time' },
-              { type: 'attribute', name: 'label', value: 'Time' },
-              { type: 'attribute', name: 'placeholder', value: '' },
-              { type: 'attribute', name: 'help-text', value: 'value is restricted to time format' }
-            ],
-            args
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'url' },
-              { type: 'attribute', name: 'label', value: 'URL' },
-              { type: 'attribute', name: 'name', value: 'url field' },
-              { type: 'attribute', name: 'placeholder', value: 'https://www.union-investment.de/' },
-              { type: 'attribute', name: 'help-text', value: 'validate with url format' }
-            ],
-            args
-          })}
-        </div>
+export const Type = {
+  render: () =>
+    html` <div class="grid grid-cols-2 gap-12 content-end">
+      <div class="mb-2">
```

Comment:
The divs around all sd-selects can be removed

Reply:
updated.